src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -185,6 +219,9 @@ def set_opts(self, curl, url, params, headers,
             else:
                 logging.warning("Wrong data type for header 'Accept-Encoding': %s",
                                 type(headers["Accept-Encoding"]))
+        else:
+            # add gzip encoding by default
+            curl.setopt(pycurl.ENCODING, 'gzip')
```

Comment:
Do you see any use case and/or scenario where hard-coding this `gzip` accept-encoding could be a problem for us?

Reply:
no, since it is up to a server to accept it or not. If accepted and returned back as gzip we have code in place to process it. Otherwise, the change will have no effect on our code.